service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionServiceTest.java

```diff
@@ -79,103 +90,88 @@
 
     @BeforeEach
     void setup() {
-        // required for DI
         given(userDetailsService.getUserName(any())).willReturn("Emma Taylor");
 
-        this.caseDataExtractionService = new CaseDataExtractionService(dateFormatterService,
-            hearingBookingService, hmctsCourtLookupConfiguration, ordersLookupService, commonDirectionService,
-            hearingVenueLookUpService, commonCaseDataExtraction);
+        this.caseDataExtractionService = new CaseDataExtractionService(hearingBookingService,
```

Comment:
If you add the LookupTestConfig to the context config you may not need to do this initialisation as court lookup could then be autowired in successfully